README.md

```diff
@@ -2,4 +2,131 @@
+* Full SDK initialised with both architectures supported by Flatcar (amd64 and arm64). This is the largest container, it's about 8GB in size (~3 GB compressed).
+* AMD64 SDK initialised for building AMD64 OS images. About 6GB in size (2GB compressed).
+* ARM64 SDK initialised for building ARM64 OS images on AMD64 hosts. Also about 6GB in size.
+While work on a native ARM64 native SDK is ongoing, it's unfortunately not ready yet. If you want to help, patches are welcome!
+
+The container can be run in one of two ways - "standalone", or integrated with the [scripts](https://github.com/flatcar-linux/scripts) repo:
+* Standalone mode will use no host volumes and will allow you to play with the SDK in a sandboxed throw-away environment. In standalone mode, you interface with Docker directly to use the SDK container.
+* Integrated mode will closely integrate with the scripts repo directory and bind-mount it as well as the portage-stable and coreos-overlay gitmodules into the container. Integrated mode uses wrapper scripts to interact with the SDK container. This is the recommended way for developing patches for Flatcar.
+
+## Standalone mode
+
+In standalone mode, the SDK is just another Docker container. Interaction with the container happens via use of `docker` directly. Use for experimenting and for throw-away work only, otherwise please use integrated mode (see below).
+
+* Check the list of available versions and pick a version to use. The SDK Major versions correspond to Flatcar Major release versions.
+  List of images: `https://github.com/orgs/flatcar-linux/packages/container/package/flatcar-sdk-all`
+  For the purpose of this example we'll use version `3033.0.0`.
+* Fetch the container image: `docker pull ghcr.io/flatcar-linux/flatcar-sdk-all:3033.0.0`
+* Start the image in interactive (tty) mode: `docker run -ti ghcr.io/flatcar-linux/flatcar-sdk-all:3033.0.0`
+  You are now inside the SDK container (the hostname will likely differ):
+  `sdk@f236fda982a4 ~/trunk/src/scripts $`
+* Initialise the SDK in self-contained mode. This needs to be done once per container and will check out the scripts, coreos-overlay, and portage-stable repositories into the container.
+  `sdk@f236fda982a4 ../sdk_init_selfcontained.sh`
+
+You can now work with the SDK container.
+
+### Privileged mode when building images
+
+In order to build OS images (via `./build_image` and `./image_to_vm`) the SDK tooling requires privileged access to `/dev`.
+This is necessary because the SDK currently employs loop devices to create and to partition OS images.
+
+To start a container in privileged mode with `/dev` available use:
+* `docker run -ti  --privileged -v /dev:/dev ghcr.io/flatcar-linux/flatcar-sdk-all:3033.0.0`
+
+## Integrated mode
+
+This is the preferred mode of working with the SDK.
+Interaction with the container happens via wrapper scripts from the Scripts repository.
```

Comment:
```suggestion
Interaction with the container happens via wrapper scripts from the scripts repository.
```

Reply:
I think podman support should be a follow-up PR (you're rightfully mentioned this one is growing big).
One key benefit of this work is that it coexists with the traditional SDK - improvements / features can be added incrementally.
Thanks for reviewing!